ehitchhiking-ui/src/app/app-routing.module.ts

```diff
@@ -1,8 +1,14 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import {MenuComponent} from './shared/components/menu/menu.component';
+import {MainScreenComponentComponent} from './main-screen/main-screen-component/main-screen-component.component';
 
 
-const routes: Routes = [];
+const routes: Routes = [
+  { path: 'menu', component: MenuComponent },
```

Comment:
Why do you add rout to the menu component? It isn't a page, it is just a component

Reply:
fixed